### PR TITLE
Applying rebrand on create team form

### PR DESCRIPTION
**Pull request by devkonrad**
Applying rebrand on create team form

Changes:
modified: app/views/student/teams/edit.html.erb
modified: app/views/student/teams/new.html.erb
modified: app/views/teams/rebrand/_form.html.erb

Refs: https://github.com/Iridescent-CM/technovation-app/issues/3453

### CODE REVIEW

**Comment by dboyer** [comment]
No issues found during functionality review. This is ready for code review.

File: app/views/teams/rebrand/_form.html.erb [hunk 1]
```diff
@@ -27,9 +27,15 @@
 
                 <p class="p-0 m-0">
                     <%= 
-                        link_to t("views.application.cancel"),
-                        send("#{current_scope}_team_path", @team),
-                        class: "tw-green-btn float-left"
+                        if @team.id.present?
```
[the remark this answers is not in the document]
Comment (devkonrad) [reply app/views/teams/rebrand/_form.html.erb:30]: Hi, Vivian! As i can remember this form is shared between "Team Creation" and "Team Update". In the "Team Creation" case, we will have the team.id, only after save the data. ;)

[the remark this answers is not in the document]
Comment (viviancan) [reply app/views/teams/rebrand/_form.html.erb:30]: Got it, thanks!

**Review by viviancan — APPROVED** [review]
LGTM 👍 Thanks Alex! I did try this locally and it worked as expected.

**Pull request merged**